azure-tools-common/src/main/java/com/microsoft/azure/common/function/handlers/AnnotationHandlerImpl.java

```diff
@@ -84,10 +92,25 @@ public FunctionConfiguration generateConfiguration(final Method method) {
 
         patchStorageBinding(method, bindings);
 
+        config.setRetry(getRetryConfigurationFromMethod(method));
         config.setEntryPoint(method.getDeclaringClass().getCanonicalName() + "." + method.getName());
         return config;
     }
 
+    private Retry getRetryConfigurationFromMethod(Method method) throws AzureExecutionException {
+        final FixedDelayRetry fixedDelayRetry = method.getDeclaredAnnotation(FixedDelayRetry.class);
+        final ExponentialBackoffRetry exponentialBackoffRetry = method.getDeclaredAnnotation(ExponentialBackoffRetry.class);
+        if (!Objects.isNull(fixedDelayRetry) && !Objects.isNull(exponentialBackoffRetry)) {
+            throw new AzureExecutionException(MULTI_RETRY_ANNOTATION);
```

Comment:
remove else

Reply:
No need to evaluate conditions below in this method, so if else may be better?

Reply:
the behaviors are same, but is more readable:https://stackoverflow.com/questions/2681883/unnecessary-else-statement

Reply:
Fixed